contrib/mixin/mixin.libsonnet

```diff
@@ -1287,7 +1287,7 @@
             name: 'cluster',
             options: [],
             query: 'label_values(etcd_server_has_leader, job)',
-            refresh: 1,
+            refresh: 2,
```

Comment:
Please add a comment explaining what does the constant mean.

Reply:
@ptabor do you mean a inline comment on the code or here? On the PR description I've detailed the change:

Reply:
In-line please. It requires a lot of knowledge to decode the constant when scanning the config.

Reply:
@ptabor great! I've create a const and set it as value 2, please let me know if it fits better. https://github.com/etcd-io/etcd/pull/12823/commits/048b90f7a7987eaa6bdd1912f46cd202c71936a6